src/utilities/replay/replay.cpp

```diff
@@ -156,6 +156,66 @@ void trim(std::string &s)
              s.end());
 }
 
+conduit::Node load_actions(const std::string &file_name, int mpi_comm_id){
+  conduit::Node actions;
```

Comment:
We prefer passing in a ref to the node and having node filled out instead of returning a node.

This is b/c returning a node involves more copies.